kpatch-build/kpatch-elf.c

```diff
@@ -332,6 +336,22 @@ static void kpatch_create_rela_list(struct kpatch_elf *kelf,
 				      rela->sym->name, rela->addend);
 		}
 
+		if (kelf->arch == LOONGARCH64) {
+			/*
+			 * LoongArch GCC creates local labels such as .LBB7266,
+			 * replace them with section symbols.
+			 */
+			if (rela->sym->sec && (rela->sym->type == STT_NOTYPE) &&
+				(rela->sym->bind == STB_LOCAL)) {
+				log_debug("local label: %s -> ", rela->sym->name);
+
+				rela->addend += rela->sym->sym.st_value;
+				rela->sym = rela->sym->sec->secsym;
+				log_debug("section symbol: %s\n", rela->sym->name);
+			}
+		}
```

Comment:
Just curious, which kernel source files are generating these symbols?

Reply:
Okey, I will try to find it.